backend/common/src/main/java/ai/verta/modeldb/common/CommonDBUtil.java

```diff
@@ -387,15 +387,17 @@ public void runMigrations(
       throws SQLException, MigrationException {
     RdbConfig rdbConfiguration = config.getRdbConfiguration();
     createDBIfNotExists(rdbConfiguration);
-    Connection connection = acquireDatabaseConnection(config, rdbConfiguration);
-    Migrator migrator =
-        new Migrator(
-            connection,
-            findResourcesDirectory(migrationResourcesRootDirectory, rdbConfiguration),
-            rdbConfiguration);
-    boolean liquibaseTableExists = tableExists(connection, config, "database_change_log");
-    migrator.preInitializeIfRequired(liquibaseTableExists, currentVersion);
-    migrator.performMigration();
+    try (Connection connection = acquireDatabaseConnection(config, rdbConfiguration)) {
```

Comment:
closing the connection seems good. I don't think we should need to be doing a commit, though.